### DISCUSSION TITLE
knowledgerails project

### DISCUSSION THREAD

**Discussion by ghost** [protocol/research#41] [Ideas, Open Problems, and Proposals]
Hello: 
I'm working on the use of crypto-currency to support scholars ex-post.
Project is early stage, but feedback welcome. 
Thanks

**Comment by jsoares** [comment]
Are you familiar with https://sourcecred.io/? Despite having been created as a way to recognise open-source contributions, the framework is way more general. We have thought of ways to apply it to research communities, including e.g. pre-paper discussions in a forum, but have nothing too concrete to show at this point.

**Reply by ghost** [reply]
yes I've heard of it, and its on my study list.
